subsys/bluetooth/controller/ll_sw/openisa/hal/cpu.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2016 Nordic Semiconductor ASA
+ * Copyright (c) 2016 Vinayak Kariappa Chettimada
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+static inline void cpu_sleep(void)
+{
+	__WFE();
+	/* __SEV(); */
```

Comment:
__SEV is used to ensure second __WFE does not wait. Assume the function entered with event flag set, then first __WFE will clear and pass through, without __SEV() the second __WFE will wait.

Reply:
ext/hal/openisa/vega_sdk_riscv/RISCV/core_riscv32.h
```
__attribute__((always_inline)) __STATIC_INLINE void __WFE(void)
{
}

```

Reply:
We'll have a look at this, and see how we can get a similar behavior as for ARM cores

Reply:
sorry... my fault... of course these instructions are not applicable for RISCV